docs/docs/development/setup/zmk.md

```diff
@@ -0,0 +1,128 @@
+---
+title: ZMK Sources
+sidebar_label: ZMK Sources
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const OsTabs = (props) => (<Tabs
+groupId="operating-systems"
+defaultValue="docker"
+values={[
+{label: 'VS Code & Docker', value: 'docker'},
+{label: 'Debian/Ubuntu', value: 'debian'},
+{label: 'macOS', value: 'mac'},
+{label: 'Windows', value: 'win'},
+{label: 'Raspberry OS', value: 'raspberryos'},
+{label: 'Fedora', value: 'fedora'},
+]
+}>{props.children}</Tabs>);
+
+### Source Code
+
```

Comment:
A note on performance with Docker on Windows would be nice, though not necessary. You will see dramatically reduced build times if you clone ZMK to the WSL2 file system, but then you have to type `\\wsl$` into File Explorer to access those files. This also requires that you install https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl